src/libs/SearchAutocompleteUtils.ts

```diff
@@ -131,6 +132,15 @@ function getAutocompleteQueryWithComma(prevQuery: string, newQuery: string) {
     return newQuery;
 }
 
+function workletizedParser(input: string) {
```

Comment:
Just noticed that this name is rather weird sounding, so I hope we can change it.